server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -278,4 +285,18 @@ export class TenantManager {
 
         await collection.update({ _id: tenantId }, { disabled: true }, null);
     }
+
+    private encryptAccessInfo(externalStorageData: any): any {
```

Comment:
Rather than passing the whole externalStorageData, can we just pass the accessInfo object? Both for encrypt and decrypt?

Reply:
Sure, will change the code in the next revision